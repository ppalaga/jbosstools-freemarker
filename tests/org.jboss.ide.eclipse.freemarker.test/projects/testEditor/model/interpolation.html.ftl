<h1>Welcome ${user}!</h1>
<p style="font-weight: bold">Our latest product:</p>
<a href="${url}">${content}<!-- comment 1 --></a>
<${tag} ${attribute}="${value}">${content}</${tag}>
<${tag} <#list attributes as attrib>${attrib.key}="${attrib.value}" </#list>>${content}</${tag}>!
<!-- <p style="font-weight:bold">Our latest product:</p> -->