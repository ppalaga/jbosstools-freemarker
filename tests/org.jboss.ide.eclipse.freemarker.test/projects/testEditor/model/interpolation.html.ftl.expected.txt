<h1>Welcome Joe!</h1>
<p style="font-weight: bold">Our latest product:</p>
<a href="http://example.com">Example Inc.<!-- comment 1 --></a>
<a href="https://example.com">Example Inc.</a>
<a href="https://example.com" target="_blank" >Example Inc.</a>!
<!-- <p style="font-weight:bold">Our latest product:</p> -->